static/app/views/settings/project/projectKeys/details/loaderSettings.tsx

```diff
@@ -211,12 +214,58 @@ export function LoaderSettings({keyId, orgSlug, project, data, updateData}: Prop
             }
             help={
               !sdkVersionSupportsPerformanceAndReplay(data.browserSdkVersion)
-                ? t('Only available in SDK version 7.x and above')
+                ? tct(
+                    `Only available in SDK version 7.x and above [br]We now support [code:canvas] recording in SDK version 7.94.1. [link:Learn more]`,
+                    {
+                      canvas: <code />,
+                      br: <br />,
+                      link: (
+                        <ExternalLink
+                          onClick={() => {
+                            trackAnalytics('replay.canvas-loader-clicked', {
+                              organization,
+                            });
+                          }}
+                          href="https://docs.sentry.io/platforms/javascript/session-replay/#canvas-recording"
+                        />
+                      ),
+                    }
+                  )
                 : data.dynamicSdkLoaderOptions.hasReplay
-                ? t(
-                    'When using Replay, the loader will load the ES6 bundle instead of the ES5 bundle.'
+                ? tct(
+                    `When using Replay, the loader will load the ES6 bundle instead of the ES5 bundle [br]We now support [code:canvas] recording in SDK version 7.94.1. [link:Learn more]`,
```

Comment:
```suggestion
                    `When using Replay, the loader will load the ES6 bundle instead of the ES5 bundle. [br]We now support [code:canvas] recording in SDK version 7.94.1. [link:Learn more]`,
```